pkg/odo/cli/component/watch.go

```diff
@@ -91,7 +91,7 @@ var watchCmd = &cobra.Command{
 func NewCmdWatch() *cobra.Command {
 	// ignore git as it can change even if no source file changed
 	// for example some plugins providing git info in PS1 doing that
-	watchCmd.Flags().StringSliceVar(&ignores, "ignore", []string{".*\\.git.*"}, "Files or folders to be ignored via regular expressions.")
```

Comment:
Why is the git part ignored? And if we are including it for some reason, please modify the comment above

Reply:
This was changed by accident, seems that rebasing caused this error.